identifiers/country-hk/geographical_constituencies.csv

```diff
@@ -1,5 +1,5 @@
 id,name
-ocd-division/country:hk/geographical_constituency:hk_island,Hongkong island
+ocd-division/country:hk/geographical_constituency:hk_island,Hong Kong island

```

Comment:
Hmm, Island is still not capitalized, but should be. For my previous suggestions, you could have clicked "Commit suggestion".

Reply:
I should have just clicked "Commit suggestion", didn't realize I could do that until after I made the edits.